style.css

```diff
@@ -0,0 +1,62 @@
+.toolbar {
+  display: flex;
+  justify-content: space-between;
+}
+
+#img1 {
+  justify-content: flex-start;
+}
+
+.ic_icon {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  flex: 0;
+  padding: 8%;
+}
+
+#btn {
+  justify-content: space-between;
+  border-radius: 8%;
+  background: #1a2236;
+}
```

Comment:
- [ ] Nice work implementing the hamburger menu :clap: However it would be nice to stick to the Figma design as much as possible. Therefore, I suggest you remove the hamburger's bar border and increase their width to follow the Figma design.
### Your design :point_down: 
![Screenshot from 2023-07-06 09-10-12](https://github.com/amel129/Setup_and_mobile_version_skeleton/assets/91142432/0ad07162-3afa-4602-9b54-3b9e6b85450b)
### What it's supposed to look like :point_down: 
![Screenshot from 2023-07-06 09-09-30](https://github.com/amel129/Setup_and_mobile_version_skeleton/assets/91142432/ce760209-fb80-4cf3-8ec8-253138711b9e)

**HIT:**
```css
#btn {
  justify-content: space-between;
  border-radius: 8%;
  background: #1a2236;
  border: none;
  width: 20px;
}
```